mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1065,6 +1107,59 @@ pub async fn nexus_create(
     size: u64,
     uuid: Option<&str>,
     children: &[String],
+) -> Result<(), Error> {
+    nexus_create_internal(
+        name,
+        size,
+        uuid,
+        NexusNvmeParams::default(),
+        children,
+    )
+    .await
+}
+
+/// As create_nexus with additional parameters:
+/// min_cntlid, max_cntldi: NVMe controller ID range when sharing over NVMf
+/// resv_key: NVMe reservation key for children
+pub async fn nexus_create_v2(
```

Comment:
Will this include the changes we spoke about in the call the other day about each nexus having a unique uuid and a separate uuid for the namespace? Or will that be part of a separate PR?
Can/Should the control plane make use of this new method for non ANA volumes?

Reply:
No, the UUID changes are not part of this ticket, it will have to be part of a separate ticket (has one been raised).
The new method can be used as long as `NexusNvmeParams` contains default values as that's what the existing method does.